src/bootstrap/src/core/build_steps/check.rs

```diff
@@ -89,7 +89,7 @@ impl Step for Std {
         let stage = self.custom_stage.unwrap_or(builder.top_stage);
 
         let target = self.target;
-        let compiler = builder.compiler(stage, builder.config.build);
+        let compiler = builder.compiler(stage, builder.config.host_target);
```

Comment:
Suggestion [NOMENCLATURE 1/2]: can we rename this instead to sth more like `host_platform` or `host_tuple`? Because `host_target` I find is still somewhat of a "mind-bender" -- what do you call the target platform, then, the "target target"? 😆

Reply:
It essentially means "target of your computer". We already use the term "target" for the "target tuple" in bootstrap, so I thought I'd stay consistent with that. And yes, we do call a few things in bootstrap "target targets" :joy:

`host_tuple` sounds fine, but it would also be the first occurrence of the word "tuple" in bootstrap :man_shrugging: